src/entity/opportunities/types.ts

```diff
@@ -1,14 +1,14 @@
 export enum OpportunityType {
-  Job = 'job',
+  Job = 'JOB',
 }
 
 export enum OpportunityUserType {
   Recruiter = 'recruiter',
 }
 
 export enum OpportunityMatchStatus {
-  Pending = 'pending',
-  CandidateAccepted = 'candidate_accepted',
+  Pending = 'PENDING',
+  CandidateAccepted = 'CANDIDATE_ACCEPTED',
```

Comment:
@omBratteng not sure if you adjusted these already in your PR to match the proto ones?
Or even better just use the proto ones directly? (maybe reuse my protoToX function?